arlo-client/src/components/DataEntry/BallotAudit.tsx

```diff
@@ -169,32 +178,33 @@ const BallotAuditContest = ({
           () => undefined
         } // required by blueprintjs but we're implementing on BlockRadio instead
         selectedValue={
-          (isVote ? interpretation.choiceId : interpretation.interpretation) ||
-          undefined
+          (isVote
+            ? interpretation.choiceIds.length
```

Comment:
what does this do here? does it make sense to use RadioGroup now that we're using checkboxes?

Reply:
@jonahkagan I initially left it in just in case removing it messed up styling or Formik, and then forgot to try removing it to see what happens. It appears to still work perfectly without it, though, so that's awesome.